cmd/lk/agent.go

```diff
@@ -43,17 +41,17 @@ const (
 )
 
 var (
-	nameFlag = func(required bool) *cli.StringFlag {
+	idFlag = func(required bool) *cli.StringFlag {
 		return &cli.StringFlag{
-			Name:     "name",
-			Usage:    fmt.Sprintf("`NAME` of the agent. If unset, and the %s file is present, will use the name found there.", config.LiveKitTOMLFile),
+			Name:     "id",
+			Usage:    fmt.Sprintf("`ID` of the agent. If unset, and the %s file is present, will use the id found there.", config.LiveKitTOMLFile),
 			Required: required,
 		}
 	}
 
-	nameSliceFlag = &cli.StringSliceFlag{
-		Name:     "name",
-		Usage:    "`NAMES` of agent(s)",
+	idSliceFlag = &cli.StringSliceFlag{
+		Name:     "id",
+		Usage:    "`IDs` of agent(s)",
 		Required: false,
 	}
```

Comment:
Do we want to preserve but deprecate the `name` flag and mark it with `Hidden: true`? Or are we just ripping the bandaid on this one since it's in beta?

Reply:
What does `Hidden: true` do? It might get confusing since when user sets a name flag and notices it does nothing.

Reply:
What does `Hidden: true` do? It might get confusing since when user sets a name flag and notices it does nothing.

Reply:
Oh, I meant keeping both sets of logic around for a while and actually handling the `name` parameter. But we don't need to do that.

`Hidden: true` removes the flag from the `--help` commands and other places so it isn't shown to users, but still works.

Reply:
Ah, gotcha. The Agents API will reject any requests that depend on agent names and return an error message asking users to upgrade livekit-cli.